uaclient/entitlements/tests/test_cc.py

```diff
@@ -49,6 +50,33 @@
 })
 
 
+class TestCommonCriteriaEntitlementOperationalStatus:
+
+    @pytest.mark.parametrize(
+        'arch,series,details',
+        (('arm64', 'xenial', 'Canonical Common Criteria EAL2 Provisioning is'
+          ' not available for platform arm64.\nSupported platforms are:'
+          ' x86_64, ppc64le, s390x\n'),
+         ('s390x', 'trusty', 'Canonical Common Criteria EAL2 Provisioning'
+          ' is not available for Ubuntu trusty.')))
```

Comment:
Why does one of these have a trailing newline and the other doesn't? Do we have inconsistent output for these two cases?

Reply:
good point, it is inconsistent spacing on the output for those cases. it probably should be the same. I'll drop the trailing newline